core/test/pick_ur5.cpp

```diff
@@ -89,12 +90,10 @@ TEST(UR5, pick) {
 
 int main(int argc, char** argv) {
 	testing::InitGoogleTest(&argc, argv);
-	ros::init(argc, argv, "ur5");
-	ros::AsyncSpinner spinner(1);
-	spinner.start();
+	rclcpp::init(argc, argv);
 
 	// wait some time for move_group to come up
-	ros::WallDuration(5.0).sleep();
+	rclcpp::sleep_for(std::chrono::seconds(5));
```

Comment:
maybe personal preference, but I think chrono literals operators just look nicer and we should use them more often.